packages/server/lib/open_project.js

```diff
@@ -55,8 +56,16 @@ const moduleFactory = () => {
     },
 
     changeUrlToSpec (spec) {
-      return openProject.getSpecUrl(spec.absolute, spec.specType)
-      .then((newSpecUrl) => openProject.changeToUrl(newSpecUrl))
+      const newSpecUrl = getSpecUrl({
```

Comment:
It called inside an `async` function. 

![image](https://user-images.githubusercontent.com/19196536/124839845-ef893080-dfcc-11eb-8b83-3930f2396966.png)

That said, if you follow this code all it does is:

```
  changeToUrl (url) {
    return this.toRunner('change:to:url', url)
  }
```

There is nothing to wait for - this code was very misleading.

The only reason it was returning a promise is because it was using `getConfig`:

![image](https://user-images.githubusercontent.com/19196536/124839966-324b0880-dfcd-11eb-9a40-7f482871d4e5.png)


Which is no longer async - it never should really should have been in the first place.